internal/testing/testing.go

```diff
@@ -0,0 +1,24 @@
+package testing
+
+// TInterface is a *testing.T interface for using ginkgo/etc with dismock
```

Comment:
```suggestion
// TInterface is a *testing.T interface for using ginkgo/etc with dismock.
```

Add a period at the end of the sentence